src/dotnet-interactive.IntegrationTests/LoggingTests.cs

```diff
@@ -86,5 +86,83 @@ public async Task kernel_server_honors_log_path()
             var logFileContents = File.ReadAllText(logFile.FullName);
             logFileContents.Should().Contain("CodeSubmissionReceived: 1+1");
         }
+
+        [IntegrationFact]
+        public async Task Pound_r_nuget_should_produce_full_log()
```

Comment:
Let's not integration test our logging. This is going to be fragile and non-critical.

Reply:
How do I test logging spew?

Reply:
We typically don't test diagnostic log output, though if it's worth doing, we can do it in-process in a more robust way. Let's chat about it later. I'd prefer to unblock this PR in the meantime.